web/packages/extension/src/content.ts

```diff
@@ -65,6 +65,7 @@ function injectScriptURL(url: string): Promise<void> {
         script.addEventListener("error", (e) => reject(e));
     });
     script.src = url;
+    script.charset = "utf-8";
```

Comment:
Actually with the escaping in `bundle_text.js`, I think this change is no longer needed :)